routes/per_capita_consumption.py

```diff
@@ -2,15 +2,15 @@
 from flask import Blueprint, jsonify
 from flask_api import status
 from flask_jwt_extended import get_jwt_identity
-from models.pkv import PKV
+from models.pcc import PerCapitaConsumption
 from models.user import User
 from util.database import db
 from util.error import UNKNOWN_USER, NO_PER_CAPITA_CONSUMPTION, exception_message
 from util.login import login_required
 
 
 logger = logging.getLogger(__name__)
-PerCapitaConsumption = Blueprint('PerCapitaConsumption', __name__)
+ShowPerCapitaConsumption = Blueprint('ShowPerCapitaConsumption', __name__)
```

Comment:
I'm not really happy with the naming here. I understand why you need to differentiate between Blueprint and column name, but what about an underscore for instance? "ShowPerCapitaConsumption" would be more of a function name than a route name, especially if you compare it to the other routes.

Reply:
If you are not familiar with the Python visibility concept and underscore conventions, you might take a look at this: https://dbader.org/blog/meaning-of-underscores-in-python